mlir/lib/Dialect/Tosa/IR/TosaOps.cpp

```diff
@@ -276,9 +384,9 @@ static void buildMatMulOpWithQuantInfo(OpBuilder &builder,
   }
 }
 
-/// Both the tosa.avg_pool2d and unary ops use the same UnaruOpQuantizationAttr
```

Comment:
Can you please revert the formatting differences here and elsewhere? We try to keep the diff small to help upstreaming the changes into llvm/llvm-project later.

Reply:
sure.

Reply:
Done